skimage/segmentation/active_contour_model.py

```diff
@@ -9,20 +9,26 @@
 def active_contour(image, snake, alpha=0.01, beta=0.1,
                    w_line=0, w_edge=1, gamma=0.01,
                    bc='periodic', max_px_move=1.0,
-                   max_iterations=2500, convergence=0.1):
+                   max_iterations=2500, convergence=0.1,
+                   callback=None):
     """Active contour model.
 
     Active contours by fitting snakes to features of images. Supports single
     and multichannel 2D images. Snakes can be periodic (for segmentation) or
     have fixed and/or free ends.
 
+    The input snake is a set of points that will each be repositioned such that
+    the new snake minimizes energy as described in [1]_. No points are added
+    or removed.

```

Comment:
How about 
No points are added or removed --> The number of points is constant, so make sure that the initial snake has enough points to capture the details of the final contour.